src/test/codegen/sse42-implies-crc32.rs

```diff
@@ -0,0 +1,16 @@
+// min-llvm-version: 14.0
+// needs-llvm-components: x86
```

Comment:
You'll need to do exactly the same thing here as for the assembly test.

Reply:
Done.